hig.js

```diff
@@ -92,12 +92,23 @@ function speak(phrases) {
 		v = 'Samantha';
 	} 
 	var ph = p.phrase.toString().startsWith('(') ? "..." : p.phrase;
-	say.speak(ph, v, 1, (err) => {
-  		if (err) {
-    		return console.error(err);
-  		}
-  		speak(phrases);
-	});		
+	console.log("monkeys");
+	let p1 = new Promise(
+        // The executor function is called with the ability to resolve or
+        // reject the promise
+       (resolve, reject) => {
+			setTimeout( function() {
+				resolve(say.speak(ph, v, 1, (err) => {
+					if (err) {
+						return console.error(err);
+					}
+				}
+			)
+		) // Yay! Everything went well!
+	 }, 1)}) 
+  		
+	
+	p1.then(setTimeout( function() {speak(phrases)}), 100000);
```

Comment:
Why do you put these `setTimeout` here? 
Someone needs to enlighten me, what does `speak()` do?

Is it a function that returns a promise?
Is it a function that triggers a callback?
Is it a synchronous function?

I should probably clone this whole thing first....

Reply:
Hey Timotej, sorry for repeating myself and I appreciate you taking the time to comment so much but this code is outdated as I was trying things out and figuring out what certain things did. My problem now is getting the asynchronous code to start the next promise when the narrator stops talking which is not resolved by promising because the previous promise, unless I am doing something wrong, resolves and starts the next promise before the narrator stops speaking.

Reply:
Okay, so since speak() is a callback it *should* resolve when the narrator has finished. And if it doesnt, I am probably doing something wrong.

Reply:
Thanks so much!